tests/unit/project_loader/extensions/test_utils.py

```diff
@@ -235,11 +233,56 @@ def test_extension_merge(self):
             Equals(["test-plug2", "test-plug"]),
         )
 
+    def test_extension_merge_plug_plug2(self):
+        self.assert_extensions("[plug, plug2]")
+
+    def test_extension_merge_plug2_plug(self):
+        self.assert_extensions("[plug2, plug]")
+
 
 class ExtensionMergeTest(ExtensionTestBase):
-    scenarios = [
```

Comment:
parameterization?

Reply:
maybe later, this module has too many fixtures